lib/clients/websocket.js

```diff
@@ -48,26 +46,37 @@ class WebsocketClient extends EventEmitter {
     this.socket = null;
   }
 
-  onOpen() {
-    this.emit('open');
+  _sendSubscription(type, { product_ids, channels }) {
+    const message = { type };
+
+    if (channels) {
+      message.channels = [...channels];
```

Comment:
Can we do this in `onOpen`? No real need here to create copies of them.

Reply:
I think the copies were because Sets were being used. I'll kill 'em.